### DISCUSSION TITLE
Route error

### DISCUSSION THREAD

**Discussion by mohamednbS** [livewire/livewire#6999] [1. Help] [answered]
How to solve this error : 
Illuminate\Routing\Exceptions\UrlGenerationException
Missing required parameters for [Route: download.document] [URI: download-document/{document}]. (View: C:\xampp\htdocs\projet 2023\GMAO-master\GMAO-master\GMAO\resources\views\dmdinterventions\index.blade.php)

This is the code of the page index.blade.php : 
@if (Auth::user()->role == "Administrateur")
 <td>
<a href="{{ route('download.document', ['document' => $oi->document]) }}"> le Rapport d'Intervention</a>
</td>@endif

And this is the route : 
Route::get('download-document/{document}', function ($document) {
    $path = storage_path('app/public/' . $document);  

    return response()->download($path, $document);})->name('download.document')->middleware('auth');

**Comment by joshhanley** [comment, marked as answer]
@mohamednbS this doesn't look Livewire related so going to close this. Please ask Laravel question in their Discord or GitHub repo. Thanks!
